src/desktop/apps/partner/client/contact.coffee

```diff
@@ -25,4 +25,5 @@ module.exports = class PartnerContactView extends Backbone.View
     @$('.partner-locations').html locationStrings.join("")
 
   renderAdditionalInfo: ->
+    @$('.partner-vat-info').html "VAT ID #: #{@partner.get('vat_number')}" if @partner.get('vat_number')
```

Comment:
We don't clear the `vat_number` field if a user changes their status to 'ineligible' or 'exempt', so we could end up showing a VAT number for either of those situations if a partner's VAT status changes. Should we also add a check for `vat_status` here?

Reply:
Good thought - I think I'd say it doesn't matter though. Since this information is publicly available, and since it's a legal requirement for us to display (and I'm not 100% sure when we can _not_ display, e.g. if they were previously registered and people made purchases from them, is there some requirement for a length of time it has to be displayed afterwards?), I feel like it's OK to continue to display it even if they are no longer collecting VAT. 

TL;DR doesn't feel like it's hurting to display in those cases and might be helping.

Reply:
> if they were previously registered and people made purchases from them, is there some requirement for a length of time it has to be displayed afterwards?

This point especially resonates with me. I think we should figure that out, but it shouldn't hold this work up.